securethenews/blog/models.py

```diff
@@ -56,6 +56,6 @@ def posts(self):
         posts = BlogPost.objects.live().descendant_of(self)
 
         # Order by most recent date first
-        posts = posts.order_by('-date')
+        posts = posts.order_by('-id')
```

Comment:
This is a good idea for achieving a fix with minimal changes, but I can think of a potential problem. Imagine:

1. Create "Blog Post 1", save as draft. It gets an `id` of 1 (because we are using the default behavior of auto-incrementing primary keys).
2. Create "Blog Post 2", save as draft. It gets an `id` of 2.
3. Publish "Blog Post 2". It is shown first in the list, since it is the only entry in the list.
4. Publish "Blog Post 1". Now it is shown first in the list, _even though it was published second_. This is a bug.

The problem is that the primary key is created when the object is first saved, which may not be the same as when it is first published (since it may be saved as a draft, and other posts created and/or published, before it is finally published).